src/Microsoft.TestPlatform.CrossPlatEngine/Client/Parallel/ParallelDiscoveryDataAggregator.cs

```diff
@@ -156,4 +168,9 @@ public void AggregateIsMessageSent(bool isMessageSent)
     /// <returns></returns>
     public List<string> GetSourcesWithStatus(DiscoveryStatus status)
         => DiscoveryManager.GetSourcesWithStatus(status, _sourcesWithDiscoveryStatus);
+
+    private void AggregateDiscoveredExtensions(IDictionary<string, ISet<string>> discoveredExtensions)
```

Comment:
IMO there is no need for a separate method here, I would do the init directly.

Reply:
Fixed.